src/components/forms/LeadFormRoachAssessment.js

```diff
@@ -0,0 +1,270 @@
+import React, { useState, useEffect } from 'react'
+import { Grid } from '@material-ui/core';
+import Controls from "../controls/Controls";
+import { useForm, Form } from './useForm';
+import { Typography } from '@mui/material';
+import axios from 'axios';
+
+
+export default function LeadForm({ formData }) {
+
+    const [formDisabled, setFormDisabled] = useState(true);
+    const initialFieldValues = formData;
+
+    // TODO: finish validation rules
+    const validate = (fieldValues = values) => {
+        let temp = { ...errors };
+        if ('propertyName' in fieldValues) {
+            temp.propertyName = fieldValues.propertyName ? "" : "This field is required.";
+        }
+        if ('contactName' in fieldValues) {
+            temp.contactName = fieldValues.contactName ? "" : "This field is required.";
+        }
+        if ('contactTitle' in fieldValues) {
+            temp.contactTitle = fieldValues.contactTitle ? "" : "This field is required.";
+        }
+        if ('address' in fieldValues) {
+            temp.address = fieldValues.address ? "" : "This field is required.";
+        }
+        if ('city' in fieldValues) {
+            temp.city = fieldValues.city ? "" : "This field is required.";
+        }
+        if ('state' in fieldValues) {
+            temp.state = fieldValues.state ? "" : "This field is required.";
+        }
+        if ('zip' in fieldValues) {
+            temp.zip = fieldValues.zip ? "" : "This field is required.";
+        }
+        if ('buildingNums' in fieldValues) {
```

Comment:
Can you make the Service Details a separate form? We need 25 different ones so will be easier to make them modular rather than having one large if statement block.